### PR TITLE
Updated contributing.rst

### PR DESCRIPTION
**Pull request by Anjalizi**
Rectified grammatical errors

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ There are many ways to contribute to Scrapy. Here are some of them:
 * Report bugs and request features in the `issue tracker`_, trying to follow
   the guidelines detailed in `Reporting bugs`_ below.
 
-* Submit patches for new functionality and/or bug fixes. Please read
+* Submit patches for new functionalities and/or bug fixes. Please read
```
Comment (kmike) [inline docs/contributing.rst]: I'm not a native speaker, but https://english.stackexchange.com/questions/196590/functionality-vs-functionalities-are-both-correct-and-idiomatic seems to suggest "functionality" is ok here

**Comment by Anjalizi** [comment]
As per the english grammar guidelines, I suppose functionalities is fine
but it's your take.

**Comment by Anjalizi** [comment]
Also apart from "functionality" vs "functionalities" there was a
significant grammatical error in the initial few lines.

**Comment by kmike** [comment]
@Anjalizi yeah, I think other changes are good 👍 Let's summon @cathalgarvey :)

File: docs/contributing.rst [hunk 2]
```diff
@@ -80,8 +80,8 @@ guidelines when reporting a new bug.
 Writing patches
 ===============
 
-The better written a patch is, the higher chance that it'll get accepted and
-the sooner that will be merged.
+The better written a patch is, higher is the chance that it'll get accepted and
```
Comment (cathalgarvey) [inline docs/contributing.rst]: This line wasn't, and isn't, very idiomatic English. I would rephrase as "The better a patch is written, the higher the chances that it'll get accepted, and the sooner it will be merged".

File: docs/contributing.rst (hunk 1 above)
Comment (cathalgarvey) [reply docs/contributing.rst]: Yea, either is fine I think. :)

**Comment by cathalgarvey** [comment]
Thanks @kmike and @Anjalizi - I left a comment on the last line, but the rest is great. :)

**Comment by Anjalizi** [comment]
It's fine sir. Should I edit and file a PR again?

**Comment by cathalgarvey** [comment]
Hey @Anjalizi - No need for a new PR, if you push new commits to the same branch on your repository (patch-2) then Github will automatically add them to this PR. :)

**Comment by cathalgarvey** [comment]
Thank you @Anjalizi! :)
